example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
```

Comment:
follow [pep8](https://www.python.org/dev/peps/pep-0008/) conventions and add an additional blank line